app/Http/Controllers/AuthController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\LoginRequest;
+use App\Http\Requests\RegisterRequest;
+use App\Repository\Eloquent\AuthRepository;
+
+class AuthController extends Controller
+{
+    protected $authRepository;
+
+    public function __construct(AuthRepository $authRepository)
```

Comment:
You need to learn coding to an interface and using the service container to resolve it to the concrete implementation.